frontend/src/components/SignForms/SignInForm/SignComponents/PasswordField.tsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable react/jsx-no-bind */
+import React from 'react';
+import TextField from '@mui/material/TextField';
+import { PasswordProps } from '../../SignInterfaces';
+import InputStyled from './styled';
```

Comment:
Please check all your imports and divide them into 3 groups: libs, absolute paths and relative paths (there must be an empty line between them)

Reply:
Fixed